requirements.txt

```diff
@@ -10,6 +10,7 @@ papermill
 gitpython
 black==19.10b0
 black-nb
+https://aka.ms/triton/packages/tritonclientutils-2.1.0-py3-none-any.whl
```

Comment:
what is this?

Reply:
tritonclientutils isn't yet a pip package, has to be installed from this link for now